docs/docs/developers/tutorials/writing_token_contract.md

```diff
@@ -439,7 +439,7 @@ A getter function for checking the public balance of the provided Aztec account.
 
 A getter function to compute the note hash and nullifier for notes in the contract's storage.
 
-This must be included in every contract because it depends on the storage slots, which are defined when we set up storage.
+This must be included in every contract because it depends on the note type IDs and storage slots, which are defined when we set up storage.
 
 #include_code compute_note_hash_and_nullifier /noir-projects/noir-contracts/contracts/token_contract/src/main.nr rust
```

Comment:
I think this should be removed from the tutorial as well, since it's gone from the source contract.